src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/BlockCollection.js

```diff
@@ -154,18 +155,43 @@ class BlockCollection<T: string, U: {type: T}> extends React.Component<Props<T,
         onChange(newValue);
     };
 
-    hasMaximumReached() {
```

Comment:
Not sure about htis rename. Think `hasMaximumReached` present it better as maxOccursReached but was never a fan of the `occurs` word 🙈